src/%IPM/Main.cls

```diff
@@ -2383,7 +2383,59 @@ ClassMethod UpdateLanguageExtensions(pVerbose As %Boolean = 0, pTestOnly As %Boo
 	Quit tSC
 #; These are the actual contents of %ZLANGC00 (to be added/updated)
 $$$STARTTAG
-ZPM(pArgs...) If $System.CLS.IsMthd("%IPM.Main", "Shell") { Do ##class(%IPM.Main).Shell(pArgs...) Quit } Else { Write !, "InterSystems Package Manager is not installed in this namespace." Quit }
+ZPM(pArgs...)
+ If $System.CLS.IsMthd("%IPM.Main", "Shell") { 
+ 	Do ##class(%IPM.Main).Shell(pArgs...) Quit
+ } Else {
+	Set ZPMXMLDir = ##class(%File).NormalizeFilename("ZPM.xml", ##class(%File).ManagerDirectory())
+ 	Write !, "InterSystems Package Manager(IPM) is not installed in this namespace.", !
```

Comment:
I think instead we should write out help text indicating how zpm can be enabled. Can list out namespaces where zpm exists already. Should have a new `enable` shell command which can have prompts and also modifiers to run silently ( like `all-ns` for all namespaces and with no modifiers enable for just the current namespace).